client/ayon_nuke/plugins/load/load_image.py

```diff
@@ -156,33 +160,33 @@ def update(self, container, context):
         inputs:
 
         """
-        node = container["node"]
-        frame_number = node["first"].value()
+        read_node = container["node"]
+        frame_number = read_node["first"].value()
 
-        assert node.Class() == "Read", "Must be Read"
+        assert read_node.Class() == "Read", "Must be Read"
 
         project_name = context["project"]["name"]
         version_entity = context["version"]
         repre_entity = context["representation"]
 
         repr_cont = repre_entity["context"]
 
-        file = get_representation_path(repre_entity)
+        filepath = get_representation_path(repre_entity)
```

Comment:
not related to this PR but we can replace this function as suggested below, if you want to support the loaders across different project. 
```suggestion
        filepath = self.filepath_from_context(context)
```